pkg/pillar/zedcloud/authen.go

```diff
@@ -559,6 +574,82 @@ func verifySignature(log *base.LogObject, certByte []byte, interm *x509.CertPool
 	return nil
 }
 
+func verifyx509TimeStamps(log *base.LogObject, realCert *x509.Certificate) bool {
```

Comment:
I see that x509.VerifyOptions has a CurrentTime option, meaning that you can do use the Verify to check whether a chain would be valid at a time you select.
You can do that e.g., by adding a time argument to verifySignature() and call it once a day.

Reply:
That is a confusion between my development hack, and realities of the timeDate arithmetic.  The thought is that people will deal in days better than seconds, and the function can easily do the required math to convert a "day" value in to the internal dateTime representation.

I'll fix this for the next round of comments.

Reply:
Sure, the location is fine.